.gitignore

```diff
@@ -1,3 +1,4 @@
 __pycache__/*
 */__pycache__/*
 .DS_Store
+*.swp
```

Comment:
Right!
You'll want to .gitignore pretty much any files that are auto-generated.
Certainly you'll want to ignore ones that are OS, or editor specific. Definitely do not couple your code to the tools you use to craft it. That's like glueing the woood-work decoy duck you're carving to the workbench. :)